.github/workflows/continuous-integration.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           php-version: "${{ matrix.php-version }}"
           extensions: "apcu, pdo, ${{ matrix.extension }}"
           coverage: "pcov"
-          ini-values: "zend.assertions=1"
+          ini-values: "zend.assertions=1, apc.enable_cli=1"
```

Comment:
Look for more occurrences of `zend.assertions=1` in the file to see where you might need to add this.